server/routes/profile.js

```diff
@@ -37,6 +43,12 @@ const createProfileRouter = ({ offenderService }) => {
       error: visitsError,
     } = visitsSummary;
 
+    const {
+      remainingPvo = 0,
+      remainingVo = 0,
+      error: visitsRemainingError = false,
```

Comment:
i guess if you did push down that logic you could possibly get away without the defaulting here

service layer returning  `{ error: true } | { count: number }`   🤷

Reply:
I don't think either defaults are required here. They're there for belt and braces I guess 🤷